src/utilities/api-requests.ts

```diff
@@ -1,4 +1,5 @@
 import axios from "axios";
+import { AxiosResponse, AxiosError } from "axios";
```

Comment:
Codacy found an issue: [Multiple imports from 'axios' can be combined into one.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5953345)